setup.py

```diff
@@ -35,6 +35,11 @@
         package_data={
             '': ['schemata/*', 'tests/*.ide', 'tests/*.mkv']
         },
+        entry_points={'console_scripts': [
+            'view-ebml=ebmlite.cli:view_ebml',
+            'ebml2xml=ebmlite.cli:ebml2xml',
+            'xml2ebml=ebmlite.cli:xml2ebml',
+        ]},
```

Comment:
Should these reference `ebmlite.cmdline_script.py` instead of `ebmlite.cli`? 

Maybe just change `cmdline_scripts.py` back to `cli.py`. I misunderstood how it worked before; I didn't notice the `entry_points` in `setup.py`, and I thought the filename was exposed to the user.